update/changelog.go

```diff
@@ -0,0 +1,72 @@
+package update
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"gopkg.in/errgo.v1"
+
+	githubClient "github.com/google/go-github/v45/github"
+
+	"github.com/Scalingo/cli/services/github"
+)
+
+func printRelease(release githubClient.RepositoryRelease) {
+	fmt.Printf("Changelog of the version %v\n\n", release.GetTagName())
+	fmt.Printf("%v\n\n", strings.ReplaceAll(*release.Body, "\\r\\n", "\r\n"))
+}
+
+func ShowLastChangelog() error {
+	ctx := context.Background()
+
+	client := github.New()
+
+	cliLastRelease, err := client.GetLatestRelease(ctx)
+	if err != nil {
+		return errgo.Notef(err, "fail to get last CLI release")
+	}
+	printRelease(*cliLastRelease)
+
+	return nil
+}
+
+func ShowChangelog(cacheVersion, newVersion string) error {
```

Comment:
Can you add a comment to explain what it does? I don't completely understand the use case. The related issue suggests to display the last version changelog after installing it.

Moreover it should be private.